.github/workflows/add-release-files.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  release:
+    types: [created, published, edited]
+name: Handle Release
+jobs:
+  generate:
+    name: Create release-artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the web-components branch
+        uses: actions/checkout@v2
+        with:
+          ref: wc-automate-release-files
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - run: yarn
+      - run: yarn build
+      - run: yarn build-bindings
+      - run: yarn pack
```

Comment:
The first three are pretty standard. `yarn pack` creates a gzipped tarball with all of the folders/files from `package.json.files` + other package files (License, readme, package.json) inside of a nested folder. This nested folder is important so that the structure will be preserved when yarn/npm unpacks it.